### PR TITLE
[FEATURE] move pathfinder to connector

### PR DESCRIPTION
**Pull request by sentientwaffle**
Corresponding sender changes: https://github.com/interledger/five-bells-sender/pull/56

This isn't breaking because its behind a feature flag by default.

### CODE REVIEW

**Comment by lumberj** [comment]
Update the routes here, https://github.com/interledger/five-bells-connector/blob/master/src/models/metadata.js#L9

File: src/app.js [hunk 1]
```diff
@@ -84,6 +84,7 @@ function createApp (config, ledgers) {
   koaApp.use(route.get('/pairs', pairs.getCollection))
 
   koaApp.use(route.get('/quote', quote.get))
+  koaApp.use(route.get('/quote_local', quote.getLocal))
```
Comment (lumberj) [inline src/app.js]: Why not use a query parameter `?local=true` instead?

File: src/models/quote.js [hunk 2]
```diff
@@ -197,6 +198,42 @@ function * getQuote (params, ledgers, config) {
...
+ * @param {String} params.source_account
+ * @param {String} params.source_amount
+ * @param {String} params.destination_ledger
+ * @param {String} params.destination_account
+ * @param {String} params.destination_amount
+ * @param {Object} config
+ * @returns {Quote[]}
+ */
+function * getQuotePath (params, config) {
+  // TODO cache pathfinder so that it doesn't have to re-crawl for every payment
```
Comment (emschwartz) [inline src/models/quote.js]: Can't wait for this change :) -- that'll improve the performance by a lot

**Comment by emschwartz** [comment]
LGTM

**Comment by sentientwaffle** [comment]
@lumberj: I'll add it to metadata.

Re:query param: `/quote_local` has a slightly different return type. And it doesn't really matter one way or the other -- it's only a temporary endpoint. It will no longer be needed once distance-vector routing is implemented.

**Comment by lumberj** [comment]
>  It will no longer be needed once distance-vector routing is implemented.

What is that? Maybe you can add some context into the commit message.

**Pull request merged**